dev_guide/managing_images.adoc

```diff
@@ -0,0 +1,582 @@
+= Managing Images
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+== Overview
+
+An
+link:../architecture/core_concepts/builds_and_image_streams.html#image-streams[image
+stream] comprises one or more
+link:../architecture/core_concepts/containers_and_images.html#docker-images[Docker
+images] identified by tags. It presents a single virtual view of related images,
+similar to a Docker image repository.
+
+By watching an image stream, builds and deployments can receive notifications
+when new images are added and react by performing a build or deployment,
+respectively.
+
+There are many ways you can interact with images and set up image streams,
+depending on where the images' registries are located, any authentication
+requirements around those registries, and how you want your builds and
+deployments to behave. The following sections cover a range of these topics.
+
+[[tagging-images]]
+== Tagging Images
+
+Before working with image streams and their tags, it will help to first
+understand image tags in the context of Docker generally.
+
+Docker images can have names added to them that make it more intuitive what they
+contain. When interacting directly with images using the `docker` CLI, the
+`docker tag` command essentially adds an alias to an image that can consist of
+several parts. Those parts can include:
+
+----
+<registry_host>/<user_name>/<image_name>:<tag>
+----
+
+The `<user_name>` part in the above could also refer to a
+link:../architecture/core_concepts/projects_and_users.html#projects[project] or
+link:../architecture/core_concepts/projects_and_users.html#namespaces[namespace]
+if the image is being stored in an {product-title} or Kubernetes environment.
```

Comment:
This is true only with OpenShift projects and only when integrated registry is used. With external registires, even deployed on OpenShift/Kubernetes it can wary depending on the configuration.